engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/functional/MetricsSpec.scala

```diff
@@ -49,8 +49,10 @@ class MetricsSpec extends fixture.FunSuite with Matchers with VeryPatientScalaFu
     processInvoker.invokeWithSampleData(process, data)
 
     MockService.data shouldNot be('empty)
-    val histogram = reporter.testMetrics[Histogram]("service.OK.serviceName.mockService.histogram").loneElement
-    histogram.getCount shouldBe 1
+    withClue(reporter.namedMetricsForScenario) {
```

Comment:
The clue can be different from result of `reporter.testMetrics`. Won't be a problem?